docs/xms.md

```diff
@@ -51,6 +49,13 @@ If the scavenger is disabled, `-Xms` &ge; `-Xmo`
 See [Using -X command-line options](x_jvm_commands.md) for more information about the `<size>` parameter.  
 See [Default settings for the OpenJ9 VM](openj9_defaults.md) for more about default values.
 
+### Calculating `-Xmx` for large pages
+
+For z/OS&reg; and Linux&reg; on IBM Z&reg;, specifying an object heap size that is a multiple of the page size uses another page of memory. For example, if the page size is 2 GB, setting `-Xmx2G` actually uses 4 GB of memory. To avoid using more memory, make the heap size a little smaller than an integral number of pages by subtracting at least 16 bytes from the value that you set with the `-Xmx` option. The following examples are provided:
```

Comment:
I am pretty sure this is obsolete. The 8 bytes tail padding on z platforms has been removed

Reply:
FYI Object Heap Tail Padding (allocation slightly more memory than requested) is done by performance improvement reason if `-Xaggressive` option is provided for all platforms. If size of physical page is smaller than 1GB one extra memory page is allocated. If size of physical page is equal or larger than 1GB no extra memory is allocated.

Reply:
>I am pretty sure this is obsolete.

Yes you are right, I didn't spot the tagging. Removing this.